modules/gapi/doc/20-kernel-api.markdown

```diff
@@ -47,7 +47,7 @@ an external parameter.
 G-API provides a macro to define a new kernel interface --
 G_TYPED_KERNEL():
 
-@snippet modules/gapi/samples/kernel_api_snippets.cpp filter2d_api
+@snippet samples/cpp/tutorial_code/gapi/kernel_api_snippets/kernel_api_snippets.cpp filter2d_api
```

Comment:
> kernel_api_snippets

I don't think we need an individual directory for every of the snippet files. You can place them all in just `doc_snippets` directory and make it once. The `tutorial_code/gapi` directory now may look overloaded with details.

Reply:
Ok, will move all snippets files to tutorial_code/gapi/doc_snippets